.bit/responses/1.6-Week 1 Step 6.md

```diff
@@ -35,58 +35,74 @@ To make a request to the CataaS API, you can try using the `node-fetch` module,
 <summary>:exclamation: How can I make a request to the CataaS API?</summary>
     </br>
 
-1. Let's use the `node-fetch` module for this task. Install it on your Azure Portal console:
+Let's use the `node-fetch` module for this task. Install it on your Azure Portal console:
 
-**Step 1: Install the module**
-In the left tab, scroll down to **Console**.
+1. Install the module: 
+    
+    In the left tab on Azure, scroll down to **Console** and enter these commands in order -
```

Comment:
I think @lazyplatypus mentioned that everything should take place in vscode (not in the azure portal)